tools/tune/dcblock/example_dcblock.m

```diff
@@ -36,3 +77,54 @@ function example_dcblock()
 rmpath ./../common
 
 end
+
+% Finds with iterative search parameter R for given cutoff frequency
+function R = dcblock_rval_calculate(fs, fc_target)
+
+if (fc_target / fs < 10 / 48e3 || fc_target / fs > 1000 / 48e3)
+	error("Illegal fc_target");
+end
+
+h_target = 1 / sqrt(2); % -3.01 dB
+R = 0.5;
+R_step = 0.005;
+sign = 1;
+err_prev = inf;
+w = 2 * pi * fc_target / fs;
+j = sqrt(-1);
+z = exp(j * w);
+
+% Iteration 0
+h = (1 - z^-1) / (1 - R * z^-1);
+a = abs(h);
+err_prev = (h_target - abs(h))^2;
+R = R + sign * R_step;
+
+% Do more iterations
+for n = 1 : 200
+	h = (1 - z^-1) / (1 - R * z^-1);
+	err = (h_target - abs(h))^2;
+	if (err > err_prev)
+		sign = -sign;
+		R_step = R_step / 2;
+	end
+	R = R + sign * R_step;
+	err_prev = err;
+end
+
+% Sane result?
+if R < eps || R > 1 - eps
+	error("Calculate of R iteration failed")
```

Comment:
Perhaps there is a missing semicolon at the end.

Reply:
@singalsu issue or not?

Reply:
I think it's good to have the semicolon there, Matlab syntax does not require it, but possibly it could display unnecessary return status from error(). I'll do Sriram's suggested fixes, stay tuned.